<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxisControlWidget</class>
 <widget class="QWidget" name="AxisControlWidget">
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="container">
     <property name="title">
      <string>Axis 1</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Axis #</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="axisSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pos:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentPositionLabel">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="absoluteRadioButton">
        <property name="text">
         <string>Absolute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="relativeRadioButton">
        <property name="text">
         <string>Relative</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="valueLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="ccwButton">
        <property name="text">
         <string>◀</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cwButton">
        <property name="text">
         <string>▶</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
